x-pack/legacy/plugins/siem/public/components/ml_popover/hooks/use_job_summary_data.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useEffect, useState } from 'react';
+import { jobsSummary } from '../api';
+import { Job } from '../types';
+
+type Return = [boolean, Job[] | null];
+
+export const getSiemJobsFromJobsSummary = (data: Job[]) =>
+  data.reduce((jobs: Job[], job: Job) => {
+    return job.groups.includes('siem') ? [...jobs, job] : jobs;
+  }, []);
```

Comment:
pure and clean, love it 👍 . You can keep the word `siem` but just letting you know that personally I still try to avoid adding marketing names or company names to any parts of my code except for something like "machine learning" since that is more of a concept than a name.

However, siem is named after a type of product, so... up to you. I just avoid the word `siem` in case we ever change our product name again.

That's just a FWIW comment, no requests asked.